indexer-cli/pom.xml

```diff
@@ -91,6 +97,7 @@ under the License.
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-webapp</artifactId>
+      <scope>test</scope>
```

Comment:
PR https://github.com/apache/maven-indexer/pull/165 missed to set scope (is test)